packages/server/package.json

```diff
@@ -60,6 +61,7 @@
     "@google-cloud/monitoring": "^2.1.1",
     "@googlemaps/google-maps-services-js": "^2.6.0",
     "@tianhuil/simple-trpc": "^1.8.0",
+    "node-fetch": "^2.6.1",
```

Comment:
`simple-trpc` I couldn't make it work with `axios` and if I left `fetch´ empty it would throw errors (by default it tries to use `window.fetch` I think)